src/gt4sd/frameworks/molgx/docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
all the docs should be built and should go in the main docs, no need for separate docs